app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -48,7 +48,9 @@
         <LinearLayout
             android:layout_height="match_parent"
             android:layout_width="match_parent"
-            android:orientation="vertical">
+            android:orientation="vertical"
+            android:id="@+id/ll_user_profile"
+            android:visibility="visible">
```

Comment:
@ivary43 I don't thing we need to declare visibility( as visible) as it is the default so can you remove this?